src/utils/tx.ts

```diff
@@ -27,9 +27,10 @@ export async function signWithKeypair(
 export async function logInvocation(invocation: Promise<ContractResult<any>>) {
   console.log('invoking contract...');
   const result = await invocation;
+  console.log('Hash: ', result.hash);
```

Comment:
Is this already included in the line below?

Reply:
No, it does not log the hash only the result, if any